app/templates/page/new.hbs

```diff
@@ -1,18 +1,22 @@
 {{outlet}}
 <div id="pageButtons">
   <span class="flexSpace">
-    <button type="button" class="btn btn-info" {{action "clearText"}}>{{t 'clear.text'}}</button>
-    <button type="button" class="btn btn-success" {{action "changeFont"}}>{{t 'change.font'}}</button>
-    <button type="button" class="btn btn-warning" id="highbutton" {{action "highlight"}}>{{t 'highlight.text'}}</button>
+    {{#paper-button raised=true onClick=(action 'clearText')}}{{t 'clear.text'}}{{/paper-button}}
+    {{#paper-button raised=true onClick=(action 'highlight') id="highbutton"}}{{t 'highlight.text'}}{{/paper-button}}
   </span>
 </div>
 <div id="pageInputs">
   <span class="flexSpace">
-    {{t 'current.font'}}: <input id="fontFamily">
+    <!--  {{t 'current.font'}}: <input id="fontFamily">
     {{t 'font.size'}}: <input id="fontSize">
-    {{t 'highlighted.text'}}: <input id="highlightInput">
+    {{t 'highlighted.text'}}: <input id="highlightInput"> -->
+    {{paper-input label=(t 'current.font') value=family onChange=(action "mutFamily") icon="font_download" id="fontFamily"}}
+    {{paper-input label=(t 'font.size') value=size onChange=(action "mutSize") icon="format_size" id="fontSize"}}
+    {{paper-input label=(t 'highlighted.text') value=highlightText onChange=(action (mut highlightText)) icon="highlight" id="highlightInput"}}
   </span>
 </div>
-<div class="container">
-  <p contenteditable="true" id="textArea">complex</p>
-</div>
+
+<div class="layout-row layout-align-center-start container">
+  <img src="https://cdn.pixabay.com/photo/2018/01/03/09/09/book-3057904_960_720.png" width="80%">
```

Comment:
Copy this asset to project and use via local path instead of a relative which can be broken sooner or later.